python/src/aac/plugins/validators/_validator_findings.py

```diff
@@ -39,6 +40,14 @@ def _add_finding_with_severity(
     ) -> None:
         location = FindingLocation.from_lexeme(validation_name, lexeme)
         self.add_findings([ValidatorFinding(definition, severity, message, location)])
+        log_message = self._get_last_finding()
+
+        if severity == FindingSeverity.INFO:
+            logging.info(log_message)
+        elif severity == FindingSeverity.WARNING:
+            logging.warning(log_message)
+        elif severity == FindingSeverity.ERROR:
+            logging.error(log_message)
```

Comment:
I'm inclined to actually log outside of this module as that would allow more flexibility in how logging is done at the borders of the application (e.g. in the CLI) rather than in the internals.